ParkenDD/Colors.swift

```diff
@@ -90,4 +104,31 @@ extension UIColor {
 		}
 		self.init(red:red, green:green, blue:blue, alpha:alpha)
 	}
+
+    // MARK: - Properties on UIColor
```

Comment:
Unnecessary comment. Either make it add value or remove it.

Reply:
Deleted